lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java

```diff
@@ -573,4 +579,48 @@ public static String getPlatformDescription(String platformItem) {
         }
         return null;
     }
+
+    /**
+     * get path for an element from the plugin config location xml file
+     * we will validate the file exists as well
+     * @param pluginConfigFilePath
+     * @param elementName
+     * @return the properties file to use, or null if not found
+     */
+    public static File getFileFromLibertyPluginXml(Path pluginConfigFilePath, String elementName) {
+         String elementValue = XmlReader.getElementValue(pluginConfigFilePath, elementName);
+            if (elementValue != null) {
+                Path elementDir = Paths.get(elementValue);
+                if (elementDir.toFile().exists()) {
+                    return elementDir.toFile();
+                }
+                else {
+                    LOGGER.warning("Path specified for %s in liberty-plugin-config path %s does not exist".formatted(elementName, pluginConfigFilePath));
+                }
+            }
+            else {
+                LOGGER.warning("Element for %s in does not exist in file %s ".formatted(elementName, pluginConfigFilePath));
```

Comment:
This doesn't read right. Remove `for` and `in`. Change to "Element %s does not exist in file %s."

Reply:
Done